share/examples/bitcoin.conf

```diff
@@ -71,6 +71,12 @@
 # configuration option or the addnode RPC, which have a separate limit of 8 connections.
 #maxconnections=
 
+# Maximum upload bandwidth target in MiB per day (e.g. 'maxuploadtarget=1024' is 1 GiB max throughput per day).
+# This limits the upload bandwidth for those with bandwidth limits. 0 = no limit (default: 0).
+# Limit does not apply to peers with 'download' permission.
```

Comment:
s/Limit/This limit/ or (probably better as it is more explicit): 
```suggestion
# -maxuploadtarget does not apply to peers with 'download' permission.
```

Reply:
Thanks, addressed in  947f973  by using the `-maxuploadtarget` suggestion